lib/bamboo/adapters/sendgrid_adapter.ex

```diff
@@ -156,6 +157,14 @@ defmodule Bamboo.SendgridAdapter do
   defp put_text_body(body, %Email{text_body: nil}), do: body
   defp put_text_body(body, %Email{text_body: text_body}), do: Map.put(body, :text, text_body)
 
+  # If you would like to add a replyto header to your email, then simply pass it in
+  # using the header property or put_header function like so:
+  # put_header("reply-to", "foo@bar.com")
```

Comment:
I think this comment would be better in the moduledoc at the top. Right above # Example Config

What do you think?